.github/workflows/ci.yml

```diff
@@ -28,6 +28,40 @@ jobs:
       - name: Check formatting
         run: npm run check-formatting
 
+      - name: Compare synced directories between PR and base
+        # if the label exists, any changes are fine, so no need to check
+        if: >-
+          ! contains(github.event.pull_request.labels.*.name , 'automation:sync-docs')
+        run: |
+          autogened=(
```

Comment:
I think the name is misleading. These aren't "autogened" their "synced". To be, the reference docs are "autogened".